dbms/src/Interpreters/ClusterProxy/DescribeStreamFactory.cpp

```diff
@@ -33,24 +32,29 @@ void DescribeStreamFactory::createForShard(
         const Context & context, const ThrottlerPtr & throttler,
         BlockInputStreams & res)
 {
-    for (const Cluster::Address & local_address : shard_info.local_addresses)
+    if (shard_info.isLocal())
```

Comment:
DESCRIBE query should go to all replicas of a shard.
For example, if a shard has one local and one remote replica, the query must go to both.
And if we have more than one local replicas, we must go to all of them.

Reply:
I see， I update the fix by commit c966bf0b95c63c9d1718dbcad2dbeeafc26b0800. please help review again